src/Page.js

```diff
@@ -658,14 +659,23 @@ class Page {
       const pageNavTitleHtml = this.generatePageNavTitleHtml();
       const pageNavHeadingHTML = this.generatePageNavHeadingHtml();
       this.pageSectionsHtml[`#${PAGE_NAV_ID}`]
-        = htmlBeautify(`<nav id="${PAGE_NAV_ID}" class="navbar navbar-light bg-transparent">\n`
-        + '<div class="border-left-grey nav-inner position-sticky slim-scroll">\n'
-        + `${pageNavTitleHtml}\n`
-        + '<nav class="nav nav-pills flex-column my-0 small no-flex-wrap">\n'
-        + `${pageNavHeadingHTML}\n`
-        + '</nav>\n'
-        + '</div>\n'
-        + '</nav>\n', Page.htmlBeautifyOptions);
+        = this.jsBeautify
+          ? htmlBeautify(`<nav id="${PAGE_NAV_ID}" class="navbar navbar-light bg-transparent">\n`
```

Comment:
I wonder if these calls were actually unnecessary, since `htmlBeautify` is called on the entire html file at the end of page generation.
Could remove these calls if so for better performance

Will mark the other calls that could be investigated as well 🙋‍♂